adapters/adtelligent/adtelligent.go

```diff
@@ -55,7 +55,9 @@ func (a *AdtelligentAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *
 
 	imps := request.Imp
 	request.Imp = make([]openrtb.Imp, 0, len(imps))
-
+	if request.Device != nil && request.Test == 1 {
+		request.Device.IP = getIP(request.Device.IP)
```

Comment:
Please clone the Device first before changing the IP. This is shared memory among adapters.

Reply:
better to delete this, and even move to core ?! what is the reason for sending ::1 to exchanges ?

Reply:
Sure. That sounds good to me.